apps/judicial-system/web/src/shared-components/CaseNumbers/CaseNumbers.tsx

```diff
@@ -1,26 +1,32 @@
+import { useIntl } from 'react-intl'
 import { Box, Text } from '@island.is/island-ui/core'
 import type { Case } from '@island.is/judicial-system/types'
+import { core } from '@island.is/judicial-system-web/messages'
 import React from 'react'
 
 interface Props {
   workingCase: Case
 }
 
 const CaseNumbers: React.FC<Props> = ({ workingCase }: Props) => {
+  const { formatMessage } = useIntl()
   return (
     <>
-      <Text
-        variant="h2"
-        as="h2"
-      >{`Mál nr. ${workingCase.courtCaseNumber}`}</Text>
-      <Box display="flex">
-        <Box marginRight={5}>
-          <Text fontWeight="semiBold">{`LÖKE málsnr. ${workingCase.policeCaseNumber}`}</Text>
-        </Box>
-        {workingCase.parentCase && (
-          <Text fontWeight="semiBold">{`Fyrra málsnr. Héraðsdóms ${workingCase.parentCase.courtCaseNumber}`}</Text>
-        )}
+      <Box marginBottom={1}>
+        <Text variant="h2" as="h2">{`${formatMessage(core.caseNumber)} ${
+          workingCase.courtCaseNumber
+        }`}</Text>
       </Box>
+      <Text fontWeight="semiBold">{`${formatMessage(core.prosecutor)}: ${
+        workingCase.court?.name
```

Comment:
I believe the court should not be the prosecutor. I think this is supposed to be `workingCase.prosecutor.institution.name`